src/AzureAuth.Test/AzureAuth.Test.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup Condition="$([MSBuild]::IsOSPlatform('Windows'))">
-    <TargetFramework>net5.0-windows10.0.17763.0</TargetFramework>
-    <SupportedOSPlatformVersion>7</SupportedOSPlatformVersion>
+    <TargetFramework>net6.0-windows</TargetFramework>
```

Comment:
TLDR; use `net6.0-windows10.0.19041.0` - see https://learn.microsoft.com/en-us/dotnet/standard/frameworks#os-version-in-tfms 


There's a difference between "netX-windows" and "netX-windows10.0.Y.Z". The first allows a windows app to use WinForms. The second also allows to call into Windows COM APIs, which we need for our older WAM interop.

Reply:
That was the magic version! Figuring that out from the docs would have taken me a lot longer! Thanks @bgavrilMS !

Reply:
Yep, the precedence rules and availability of different components per target is confusing. 
Aside: With our [new WAM implementation](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/wiki/WAM), you can just target plain net6.